src/main/java/core/basesyntax/dao/FruitFileWriter.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.exceptions.NullFileNameException;
+import core.basesyntax.model.Fruit;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitFileWriter implements FruitDataWriter {
+    private final List<Fruit> fruits;
+    private final String fileName;
+
+    public FruitFileWriter(List<Fruit> fruits, String fileName) {
+        this.fruits = fruits;
+        if (fileName == null) {
+            throw new NullFileNameException("Can't write in a file with NULL name");
+        }
+        this.fileName = fileName;
+    }
+
+    @Override
+    public void write() {
+        try (BufferedWriter bufferedWriter = Files.newBufferedWriter(Paths.get(fileName))) {
+            bufferedWriter.write("fruit,quantity" + System.lineSeparator());
```

Comment:
same for this header line - this is part of the report

writer should only write to file - it is its single responsibility